supabase/functions/_backend/utils/pg.ts

```
@@ -54,17 +61,48 @@ export function getDatabaseURL(c: Context): string {
+    // Connection timeout
+    connect_timeout: 10,
+  })
+
+  return cachedPgClient
 }
 
 export function getDrizzleClient(client: ReturnType<typeof getPgClient>) {
   return drizzle(client as any, { logger: true })
 }
 
 export function closeClient(c: Context, client: ReturnType<typeof getPgClient>) {
```

ESLint

[error] 'client' is defined but never used. Allowed unused args must match /^_/u.

(unused-imports/no-unused-vars)